cli/application/src/main/java/com/buschmais/jqassistant/commandline/task/RegisteredTask.java

```diff
@@ -136,12 +139,19 @@ public static List<Task> getTasks() {
         return tasks;
     }
 
-    public static List<String> getTaskNames() {
-        List<String> taskNames = new ArrayList<>();
+    public static Map<String, String> getTaskNamesAndDescriptions() {
+        Map<String, String> taskNameAndDescription = new HashMap<>();
```

Comment:
The HashMap generates a "pseudo-random" order of entries, please use a TreeMap und return this as SortedMap from the method.